CLion_Engine-Source_Linux/engine/.run/9k idle, 1k fighting.run.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="9k idle, 1k fighting" type="CMakeRunConfiguration" factoryName="Application" PROGRAM_PARAMS="test/9k_idle_1k_fighting.mission -d/home/stephen/projects/repos/git/GitHub/Assets-Production --debug=2" REDIRECT_INPUT="false" ELEVATE="false" USE_EXTERNAL_CONSOLE="false" WORKING_DIR="file://$PROJECT_DIR$/../../Assets-Production" PASS_PARENT_ENVS_2="true" PROJECT_NAME="Vega_Strike" TARGET_NAME="vegastrike-engine" CONFIG_NAME="Debug - GLVND" RUN_TARGET_PROJECT_NAME="Vega_Strike" RUN_TARGET_NAME="vegastrike-engine">
```

Comment:
I'm not familiar with these files. What do they do?

Reply:
In CLion, when you go to run or debug the code, these files determine what choices show up in the drop-down list of things to run/debug.